packages/react-checkbox/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -4,7 +4,7 @@ import { create } from '@uifabric/utilities/lib/test';
 import { mount, ReactWrapper } from 'enzyme';
 
 import { Checkbox } from './Checkbox';
-import { IRefObject, resetIds } from '../../Utilities';
+import { IRefObject, resetIds } from '@uifabric/utilities';
```

Comment:
You can import `@uifabric/styling` and `@uifabric/utilities` from `office-ui-fabric-react/lib/Styling` and `office-ui-fabric-react/lib/Utilities` respectively and remove those dependencies from the `package.json` file.

Reply:
The direct `@uifabric/styling` and `@uifabric/utilities` imports are correct here too for reasons explained in other PR.